bindata/scripts/clean-k8s-services.sh

```diff
@@ -10,7 +10,11 @@ chroot_path="/host"
 function clean_services() {
   # Remove switchdev service files
   rm -f $chroot_path/etc/systemd/system/switchdev-configuration.service
```

Comment:
this and L15 are no longer created by config daemon. however we should keep them in case of leftovers from old SR-IOV network operator that did not do the clean of this file.

can you add a note explaining why we delete file that were not created by current version of network operator ?

Reply:
Added